src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/AddingMultipleItemsListView.cs

```diff
@@ -13,43 +13,37 @@ public AddingMultipleItemsListView(TestDevice testDevice) : base(testDevice)
 
     public override string Issue => "Adding Multiple Items to a ListView";
 
-    // [Test]
-    // [FailsOnIOSWhenRunningOnXamarinUITest]
-    // public void AddingMultipleListViewTests1AllElementsPresent()
-    // {
-    //     App.WaitForElement("Big Job");
-    //     App.WaitForElement("Smaller Job");
-    //     App.WaitForElement("Add On Job");
-    //     App.WaitForElement("Add One");
-    //     App.WaitForElement("Add Two");
-    //     App.WaitForElement("3672");
-    //     App.WaitForElement("6289");
-    //     App.WaitForElement("3672-41");
-    //     App.WaitForElement("2");
-    //     App.WaitForElement("2");
-    //     App.WaitForElement("23");
-    //     App.Screenshot("All elements are present");
-    // }
-
-    // [Test]
-    // [FailsOnIOSWhenRunningOnXamarinUITest]
-    // public void AddingMultipleListViewTests2AddOneElementToList()
-    // {
-    //     App.Tap("Add One");
-
-    //     App.WaitForElement("1234", timeout: TimeSpan.FromSeconds(2));
-    //     App.Screenshot("One more element exists");
-    // }
-
-    // [Test]
-    // [FailsOnIOSWhenRunningOnXamarinUITest]
-    // public void AddingMultipleListViewTests3AddTwoElementToList()
-    // {
-    //     App.Screenshot("Click 'Add Two'");
-    //     App.Tap("Add Two");
-
-    //     App.WaitForElement("9999", timeout: TimeSpan.FromSeconds(2));
-    //     App.WaitForElement("8888", timeout: TimeSpan.FromSeconds(2));
-    //     App.Screenshot("Two more element exist");
-    // }
+	[Test]
+	public void AddingMultipleListViewTests1AllElementsPresent()
+	{
+		App.WaitForElement("Big Job");
+		App.FindElement("Big Job");
+		App.FindElement("Smaller Job");
+		App.FindElement("Add On Job");
+		App.FindElement("Add One");
+		App.FindElement("Add Two");
+		App.FindElement("3672");
+		App.FindElement("6289");
+		App.FindElement("3672-41");
+		App.FindElement("2");
+		App.FindElement("2");
+		App.FindElement("23");
+	}
+
+	[Test]
+	public void AddingMultipleListViewTests2AddOneElementToList()
+	{
+		App.Tap("Add One");
+
+		App.WaitForElement("1234");
+	}
+
+	[Test]
+	public void AddingMultipleListViewTests3AddTwoElementToList()
```

Comment:
This test is failing on iOS:
<img width="1145" alt="image" src="https://github.com/user-attachments/assets/2869a34a-8cfc-4be6-a2d9-e3b68689389a">

Line 46: `App.WaitForElement("9999");`

Reply:
Resolved and changes committed